Xamarin.Forms.Platform.iOS/CollectionView/ItemsViewController.cs

```diff
@@ -28,34 +24,29 @@ public abstract class ItemsViewController : UICollectionViewController
 		public ItemsViewController(ItemsView itemsView, ItemsViewLayout layout) : base(layout)
 		{
 			ItemsView = itemsView;
-			ItemsSource = CreateItemsViewSource();
-
-			UpdateLayout(layout);
+			ItemsViewLayout = layout;
 		}
 
-		public void UpdateLayout(ItemsViewLayout layout)
+		public void UpdateLayout(ItemsViewLayout newLayout)
```

Comment:
This method was previously being shared by `ItemsViewController` as well as `ItemsViewRenderer`. Since we moved code out of the constructor into `ViewDidLoad`, this method is now being used **only** by `ItemsViewRenderer`.